src/openforms/middleware.py

```diff
@@ -18,3 +20,18 @@ def __call__(self, request):
             value = SAMESITE_VALUE if origin and not settings.DEBUG else "Lax"
             response.cookies[settings.SESSION_COOKIE_NAME]["samesite"] = value
         return response
+
+
+class SessionTimeoutMiddleware:
+    """
+    Allows us to set the expiry time of the session based on what
+    is configured in our GlobalConfiguration
+    """
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        # https://docs.djangoproject.com/en/2.2/topics/http/sessions/#django.contrib.sessions.backends.base.SessionBase.set_expiry
+        request.session.set_expiry(GlobalConfiguration.get_solo().admin_session_timeout)
```

Comment:
if minutes this would become something like `timedelta(minutes=timeout)`

Reply:
Ya, it would be 
```
>>> timedelta(minutes=5).seconds
300
```
though we could also just multiply this number by `60`.  Not sure if one is preferred over the other?

Reply:
you'd want `total_seconds` if you convert that, which takes into account possible days/hours/minutes etc.